### DISCUSSION TITLE
RPK full suggestion

### DISCUSSION THREAD

**Discussion by ghost** [HasangerGames/suroi#374] [Ideas]
Stats:
{
            idString: "rpk",
            name: "RPK",
            ammoType: "762mm",
            ammoSpawnAmount: 135,
            capacity: 45,
            reloadTime: 3,
            fireDelay: 85,
            switchDelay: 400,
            recoilMultiplier: 0.75,
            recoilDuration: 150,
            fireMode: FireMode.Auto,
            shotSpread: 2,
            moveSpread: 5,
            length: 8,
            fists: {
                left: Vec.create(130, -2),
                right: Vec.create(30, 0),
                rightZIndex: 4,
                animationDuration: 100
            },
            image: { position: Vec.create(80, 0) },
            casingParticles: [{
                position: Vec.create(4.2, 0.4)
            }],
            gasParticles: gasParticlePresets.rifle,
            ballistics: {
                damage: 15,
                obstacleMultiplier: 1.5,
                speed: 0.27,
                range: 200,
                tracer: {
                    length: 1.5
                }
            }
        },
--------------
SVGs:
Loot
![rpk](https://github.com/user-attachments/assets/3574dd76-126e-4010-a824-9b8f139cbb70)
World
![rpk_world](https://github.com/user-attachments/assets/7c031b49-2ff5-42ae-b281-2b9f4568abe3)
Killfeed
![rpk_killfeed](https://github.com/user-attachments/assets/0acd46ff-7dd3-4a44-8d12-cacb591cf275)

**Comment by 1092384** [comment]
nvm we're not adding it

**Reply by ghost** [reply]
watch your tone

**Reply by 1092384** [reply]
ur a furry

**Reply by ghost** [reply]
please maintain a professional environment, you are going off-topic

**Reply by 1092384** [reply]
top 10 professional enviornments

**Reply by SovietNuclearFridgeBomb** [reply]
can we kiss

**Reply by 1092384** [reply]
yes

**Reply by Affrimation** [reply]
> can we kiss

>:(

**Reply by SovietNuclearFridgeBomb** [reply]
:(

**Reply by ghost** [reply]
> nvm we're not adding it
Why not?

**Reply by bien-star** [reply]
Because I said so

**Comment by emeraldn30** [comment]
💀

**Comment by 1092384** [comment]
get off github emerald

**Reply by ghost** [reply]
> get off github emerald

don't talk to my bro emerald like that

**Comment by HughJassss** [comment]
fuck no

**Comment by HughJassss** [comment]
-1002109301278 aura

**Reply by bien-star** [reply]
no lies were spoken

**Comment by HughJassss** [comment]
GET OUT
